apps/functions-subscription/src/domain/trial.ts

```diff
@@ -27,6 +29,8 @@ export const TrialCodec = t.intersection([
       CREATING: null,
       CREATED: null,
     }),
+    createdAt: IsoDateFromString,
+    updatedAt: IsoDateFromString,
```

Comment:
**note**: these fields are useful to distinguish between a creation or an update.

Since with Cosmos DB trigger is not possible to discriminate between a create or an update, we can use those fields: on the creation of a trial, the values will be the same; every time we update the trial document, we are going to change the `updatedAt`.
In this way, on every create/update on the `trials` container, the `trialConsumer` has a way to discriminate between a creation or an update